README.md

```diff
@@ -66,7 +67,9 @@ The above command will output two tables which are used in the ```annotate``` co
 
 #### **STRait Razor**
 
-If using the output from STRait Razor, the files **must** be labeled as ```SampleID_STRaitRazor.txt``` (example: ```Sample0001_STRaitRazor.txt```) and **must** be compiled in a separate folder (labeled with the project ID). The user must specify the folder name for the ```format``` command as well as an output filename (all sample files will be compiled into one file):
+If using lusSTR version 0.4 or above, STRait Razor data **must** be produced using the STRait Razor config file released in October 2020 (ForenSeqv1.25.config and PowerSeqv2.1.config). These config files are available [here](https://github.com/Ahhgust/STRaitRazor).
```

Comment:
I'd suggest using a permalink here: https://github.com/Ahhgust/STRaitRazor/tree/103ef68746f010add8f21266fa8bf8fb9f4a076e/

Reply:
Also, #37 mentions that the PowerSeq config file was last updated on 2020-10-22, but it looks like the PowerSeq config file in the Github repo was updated several days ago (2021-01-05) without updating the file version number. It's probably worth checking the extent of those changes, if they have any impact on lusSTR's behavior, and whether more specificity in the instructions is required.

Reply:
Ahh, good point! I had found an issue with one of the loci and they fixed it. I'll update to the most recent date.

Reply:
Ok, I updated the date and the link @standage!

Reply:
Could we use the permalink I posted a few comments ago?

Reply:
Oh, yes! Sorry.